Floofbot/Services/EventLoggerService.cs

```diff
@@ -73,16 +74,24 @@ private async Task HandleBadMessage(SocketUser user, SocketMessage msg)
         }
         public Task OnMessage(SocketMessage msg)
         {
+            var userMsg = msg as SocketUserMessage;
+            int argPos = 0;
             var _ = Task.Run(async () =>
             {
                 try
                 {
                     if (msg.Author.IsBot)
                         return;
                     var channel = msg.Channel as ITextChannel;
+                    string content = msg.Content;
                     bool hasBadWord = _wordFilterService.hasFilteredWord(new FloofDataContext(), msg.Content, channel.Guild.Id, msg.Channel.Id);
                     if (hasBadWord)
                         await HandleBadMessage(msg.Author, msg);
+                    if (userMsg.HasMentionPrefix(_client.CurrentUser, ref argPos) && content.EndsWith("?"))
```

Comment:
shouldn't we use else if here, because we don't want the 8ball to answer the question?

Reply:
Doesn't the `await HandleBadMessage(msg.Author, msg);` return from the function anyway?

Reply:
Added the `else if` anyway

Reply:
await is a keyword used with calling an async function. If we want to return we'd explicitly have to use the return keyword, e.g. `return await ...`.